src/main/java/io/github/thebusybiscuit/sensibletoolbox/SensibleToolboxPlugin.java

```diff
@@ -172,6 +172,7 @@
     private MinecraftVersion minecraftVersion = MinecraftVersion.UNKNOWN;
 
     private ConfigurationManager configManager;
+    private boolean slimefunEnabled = false;
```

Comment:
This will always be false, it's never set to true anywhere...

Reply:
Boo.. I had to revert some changes when I needed to fetch the 1.17 et al changes and must have missed this. Herp derp - Thanks :D